runtime/near-vm-runner/fuzz/fuzz_targets/diffrunner.rs

```diff
@@ -0,0 +1,39 @@
+#![no_main]
+
+use near_primitives::contract::ContractCode;
+use near_primitives::runtime::fees::RuntimeFeesConfig;
+use near_primitives::version::PROTOCOL_VERSION;
+use near_vm_logic::mocks::mock_external::MockedExternal;
+use near_vm_logic::VMConfig;
+use near_vm_runner::internal::VMKind;
+use near_vm_runner::VMResult;
+use near_vm_runner_fuzz::{ArbitraryModule, create_context, find_entry_point};
+
+libfuzzer_sys::fuzz_target!(|module: ArbitraryModule| {
+    let code = ContractCode::new(module.0.module.to_bytes(), None);
+    let wasmer2 = run_fuzz(&code, VMKind::Wasmer2);
+    let wasmtime = run_fuzz(&code, VMKind::Wasmtime);
+    assert_eq!(wasmer2, wasmtime);
```

Comment:
Hm, this I think *shouldn't* work? wasmer2 and wasmtime differ at least in the texts of some error messages. 

If this actually works, perhaps that means that our fuzzing is not deep at all? Makes sense to look at `.opaque_err` tests too see which cases should trip this assert.

Reply:
Well, I ran it only a few minutes on my laptop, so there's quite a chance that given a bit more time it'd have found error messages that differ! That said, looking for .opaque_error() looks to me like something that'd at least take a few hours for our fuzzing infra to find (sandbox-specific imports we probably can't generate, or too many imports), so it makes sense to me that my laptop wasn't able to find them :)
I just pushed a commit censoring the error contents, so this should hopefully be handled now :) I'm just not sure,